includes/utilities.inc

```diff
@@ -58,109 +58,39 @@ function islandora_cwrc_document_download_datastream(AbstractDatastream $datastr
 }
 
 /**
- * Wrapper for islandora_view_datastream() that modifies CWRC datastreams.
+ * Wrapper for islandora_view_datastream() that styles CWRC datastreams.
  *
  * @see islandora_view_datastream()
  */
 function islandora_cwrc_document_view_datastream(AbstractDatastream $datastream, $download = FALSE, $version = NULL) {
-  $object = $datastream->parent;
   module_load_include('inc', 'islandora', 'includes/datastream');
+  module_load_include('inc', 'islandora_cwrc_writer', 'includes/document');
 
-  if ($datastream->id == 'CWRC' && in_array('cwrc:documentCModel', $object->models)) {
```

Comment:
Ooooooone more thing haha ... we should probably put this check back in here as well; otherwise we're trying to style every datastream that ever gets downloaded or viewed.